### PR TITLE
Hint at parameter definition in task metadata

### PR DESCRIPTION
**Pull request by turbodog**

### CODE REVIEW

**Comment by DavidS** [comment]
Are the URLs here correct? I would have expected them to go to docs.puppet.com, not puppet.com/docs

**Comment by turbodog** [comment]
I updated links to docs.puppet.com

File: object_templates/task.erb [hunk 1]
```diff
@@ -6,5 +6,23 @@
 #
 # You can write Puppet tasks in any language you want and it's easy to 
 # adapt an existing Python, PowerShell, Ruby, etc. script. Learn more at:
-# http://puppet.com/docs/bolt/latest/converting_scripts_to_tasks.html 
+# http://docs.puppet.com/bolt/latest/converting_scripts_to_tasks.html
```
Comment (jbondpdx) [inline object_templates/task.erb]: the link should be https://puppet.com/docs/bolt/0.5/converting_scripts_to_tasks.html

Comment (bmjen) [reply object_templates/task.erb]: the version should still be 'latest' right @jbondpdx ?

Comment (jbondpdx) [reply object_templates/task.erb]: Oh, yes, sorry latest will work.

We'll no longer have an actual URL that *says* "latest," but latest will go to whatever the most recent version is.

File: object_templates/task.erb [hunk 2]
```diff
@@ -6,5 +6,23 @@
...
+#     "instances": {
+#       "description": "Number of instances to create",
+#       "type": "Integer"
+#     }, 
+#     "datacenter": {
+#       "description": "Datacenter where instances will be created",
+#       "type": "Enum[portland, sydney, belfast, singapore]"
+#     }
+#   }
+# Learn more at: https://docs.puppet.com/bolt/latest/task_metadata.html
```
Comment (bmjen) [inline object_templates/task.erb]: `puppet.com/docs` here too

**Pull request merged**